demos/layout/layouts_admin.php

```diff
@@ -37,7 +37,7 @@
 \Atk4\Ui\Header::addTo($layout, ['Basic Form Example']);
 
 $form = \Atk4\Ui\Form::addTo($layout, ['segment']);
-$form->setModel(new \Atk4\Data\Model());
+$form->setModel((new \Atk4\Data\Model())->createEntity());
```

Comment:
Would it be better to have Form::setModel() method to check for an entity and create an empty one if model is passed? This would allow passing a model instance as before.

Reply:
I am for as much less guessing as possible. In user action class, the guessing is already more like for BC.

My question here is - why would anyone pass model to a form, which can always represent only one entity?

Reply:
I agree with you but AFAIK there is no entity class and the method name is still setModel() not setEntity()...
I think it would be confusing the way it is. Until we can pass an Entity without being a Model, I think we should check this in setModel()

Reply:
I understand it can be confusing, but when entity should be allowed only, it is better to not allow anything else. When model is passed, very clear exception is throw and the developer should grasp very intuitively that only entity can be passed for components that operate with one record only. The same for model, when model is excepted, entity is rejected.

Reply:
Probably best changing method name then eventually, but I am ok with this now.